terraform/hashistack/policy_buildbot.tf

```diff
@@ -0,0 +1,15 @@
+data "nomad_acl_policy" "buildbot_worker_admin" {
```

Comment:
This has to be a resource to be applied, so perhaps this is missing a paired `resource` that actually applies it?